AnkiDroid/src/main/java/com/ichi2/anki/NavigationDrawerActivity.kt

```diff
@@ -348,6 +346,22 @@ abstract class NavigationDrawerActivity :
         startActivity(intent)
     }
 
+    /**
+     * Opens the Statistics Screen.
+     */
+    protected fun openStatistics() {
+        val intent = com.ichi2.anki.pages.Statistics.getIntent(this)
+        startActivity(intent)
+    }
+
+    /**
+     * Opens the Settings Screen.
```

Comment:
I felt it was a little ambiguous as we have a few settings screens

```suggestion
     * Opens AnkiDroid's Settings Screen.
     * @see Preferences
```